src/util/system.cpp

```diff
@@ -966,6 +966,30 @@ std::string ArgsManager::GetChainName() const
     return CBaseChainParams::MAIN;
 }
 
+void ArgsManager::logArgsPrefix(
+    const std::string& prefix,
+    const std::map<std::string, std::vector<std::string>>& args) const
+{
+    for (const auto& arg : args) {
+        if (FlagsOfKnownArg(arg.first) & DONT_LOG)
+            continue;
```

Comment:
What do you think? Somehow I changed my mind and thought it might be better to not show it at all. But it is more informative to show that the argument was specified. I can easily change it back.

Reply:
@jkczyz, after more thought, I agree your idea is better, 676eb23315cd23c84bbfe5bc4c41652b1f6d8bb5.